EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -50,45 +54,48 @@
     private String eventTimestamps;
     private Form layoutForm;
     protected String eventTypeName;
-    private EapEventType selectedEventType = new EapEventType("test" );
+    private EapEventType selectedEventType = new EapEventType("test");
     private ListView<TypeTreeNode> listview;
-    private HashMap<TypeTreeNode, String> attributeInput = new HashMap<>();
+    private List<AttributeInput> attributeInputs = new ArrayList<>();
     private WebMarkupContainer listContainer;
     private AttributeDependencyManager attributeDependencyManager;
     private final List<EapEventType> eventTypes = EapEventType.findAll();
 
+    private static final Logger logger = Logger.getLogger(GeneratePanel.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "logger" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)